scripts/spell-caster/src/utils/forge.ts

```diff
@@ -0,0 +1,7 @@
+import { $ } from 'bun'
+import { Address } from 'viem'
+
+export async function deployContract(contractName: string, rpc: string, from: Address): Promise<Address> {
+  const result = await $`forge create  --rpc-url ${rpc} --from ${from} ${contractName} --unlocked --json`.json()
+  return result.deployedTo
```

Comment:
```suggestion
  const result = await $`forge create --rpc-url ${rpc} --from ${from} ${contractName} --unlocked --json`.json()
```

Reply:
Uhh i don't understand this change, can you explain?

Reply:
Two spaces between create and --rpc-url